src/main/java/org/opensearch/flowframework/common/CommonValue.java

```diff
@@ -233,4 +233,12 @@ private CommonValue() {}
     public static final String CREATE_INGEST_PIPELINE_MODEL_ID = "create_ingest_pipeline.model_id";
     /** The field name for reindex source index substitution */
     public static final String REINDEX_SOURCE_INDEX = "reindex.source_index";
+
+    /*
+     * Constants associated with API specification
+     */
+
+    public static final String ML_COMMONS_API_SPEC_YAML_URI =
```

Comment:
Add a `/**` comment for the constant to make it part of the javadoc.  You can also put it on one line.  Not sure it needs its own category yet but won't hurt to leave that there for now.